doc/creating_file_importers.md

```diff
@@ -99,6 +99,11 @@ implementation of the following methods:
 6. **logo_source** Return the source of the logo displayed in choose
    importer view.
 
+7. **feature_name** return the name of the feature against which 
+   file importer will be registered in the host application.
+   feature_name can be like `<file_importer_name>_file_importer`
+
+
```

Comment:
The import does not need to know about features. The Pexel's plugin uses this method when registering the feature, but also there is could just as well be hard coded in the plugin class.

Reply:
sorry forgot to remove this